grafana-plugin/src/network/oncall-api/autogenerated-api.types.d.ts

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
```

Comment:
We shouldn't edit this file, it's autogenerated so on the next run your change will be lost. Instead we can edit `generate-types` script to do the same on every run. I suggest:
![Uploading image.png…]()